src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1250,6 +1250,7 @@ internal static partial class EncodingTable
             28599, // iso-8859-9
             28603, // iso-8859-13
             28605, // iso-8859-15
+            29001, // x-europa
```

Comment:
I have shared the tool to you at http://1drv.ms/1AGyTQ8 
you may run it at least to ensure we have everything is right. also if you modified the files there please share your modification to make sure we'll not lose this change later